test/pdal_ign_macro/test_preprocessing_mnx.py

```diff
@@ -91,8 +92,12 @@ def test_preprocess_mnx(
             tile_coord_scale=10,
         )
 
-        pipeline_output = pdal.Pipeline()
-        pipeline_output |= pdal.Reader.las(las_output.name)
+        pipeline_mtd = pdal.Reader.las(las_output.name).pipeline()
+        metadata = pipeline_mtd.quickinfo["readers.las"]["dimensions"]
+        assert dsm_dimension in metadata, "DSM marker dimension not found"
+        assert dtm_dimension in metadata, "DTM marker dimension not found"
+
+        pipeline_output = pdal.Reader.las(las_output.name).pipeline()
```

Comment:
commentaire du style
pipeline.quickinfo done before need that we re create the pipeline